ethcore/src/client/client.rs

```diff
@@ -413,10 +411,6 @@ impl Client {
 			}
 		}
 
-		if self.chain_info().best_block_hash != original_best {
-			self.miner.update_sealing(self);
```

Comment:
Why it's not required anymore? Is miner being informed in some other place as well?